velox/functions/lib/DateTimeFormatter.h

```diff
@@ -184,11 +184,13 @@ class DateTimeFormatter {
       std::unique_ptr<char[]>&& literalBuf,
       size_t bufSize,
       std::vector<DateTimeToken>&& tokens,
-      DateTimeFormatterType type)
+      DateTimeFormatterType type,
+      bool truncateFractionOfSecondToMillis)
```

Comment:
Would you please add description for this parameter, including how the fraction part is handled when true and false?